Library/Formula/git-hooks.rb

```diff
@@ -0,0 +1,10 @@
+require "formula"
+
+class GitHooks < Formula
+  homepage "https://github.com/icefox/git-hooks"
+  head "https://github.com/icefox/git-hooks.git"
```

Comment:
Use `https://github.com/icefox/git-hooks/archive/1.00.0.tar.gz` as the `url` and add a `sha1`.